lib/trento_web/openapi/v1/schema/activity_log.ex

```diff
@@ -8,35 +8,44 @@ defmodule TrentoWeb.OpenApi.V1.Schema.ActivityLog do
     %{
       title: "ActivityLog",
       description: "Activity Log for the current installation.",
-      type: :array,
-      items: %Schema{
-        title: "ActivityLogEntry",
-        type: :object,
-        additionalProperties: false,
-        properties: %{
-          id: %Schema{
-            type: :string,
-            description: "Identifier of Activity Log entry.",
-            format: :uuid
-          },
-          type: %Schema{
-            type: :string,
-            description: "Type of Activity Log entry."
-          },
-          actor: %Schema{
-            type: :string,
-            description: "Actor causing an Activity Log entry. E.g. System or a specific user."
-          },
-          metadata: %Schema{
-            type: :object
-          },
-          occurred_on: %Schema{
-            type: :string,
-            description: "Timestamp upon Activity Log entry insertion."
+      type: :object,
+      properties: %{
+        data: %Schema{
+          type: :array,
+          items: %Schema{
+            title: "ActivityLogEntry",
+            type: :object,
+            additionalProperties: false,
+            properties: %{
+              id: %Schema{
+                type: :string,
+                description: "UUID (v4) of Activity Log entry."
+              },
+              type: %Schema{
+                type: :string,
+                description: "Type of Activity Log entry."
+              },
+              actor: %Schema{
+                type: :string,
+                description:
+                  "Actor causing an Activity Log entry. E.g. System or a specific user."
+              },
+              metadata: %Schema{
+                type: :object
+              },
+              occurred_on: %Schema{
+                type: :string,
+                description: "Timestamp upon Activity Log entry insertion."
+              }
+            },
+            required: [:type, :actor, :metadata, :occurred_on]
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Maybe it's an opportunity to extract this definition?